### ISSUE TITLE
redirectWhenHttpProto generates invalid config

### ISSUE THREAD

**Issue by ignacioalles** [vfarcic/docker-flow-proxy#312]
**Description**

When using 'redirectWhenHttpProto' parameters the resulting configuration is invalid.
Relevant labels in docker-compose file are:
```
labels:
    - com.df.notify=true
    - com.df.distribute=true
    - com.df.aclName=project
    - com.df.serviceDomain=api.domain.com
    - com.df.servicePath=/users/
    - com.df.reqPathSearch=/users/
    - com.df.reqPathReplace=/api/
    - com.df.port=80
    - com.df.redirectWhenHttpProto=true
```

**Describe the results you received:**

The generated HA Proxy configuration is:
```
frontend services
    acl url_project80_0 path_beg /users/
    acl domain_project80_0 hdr(host) -i api.domain.com
    acl is_project_http hdr(X-Forwarded-Proto) http
    redirect scheme https if is_project_http url_project80_0 domain_project80
    use_backend project_apiusers_web-be80_0 if url_project80_0 domain_project80_0

backend project_apiusers_web-be80_0
    mode http
    http-request set-path %[path,regsub(/users/,/api/)]
    server project_apiusers_web project_apiusers_web:80
```

**Describe the results you expected:**

The line
`redirect scheme https if is_project_http url_project80_0 domain_project80`
should has been
`redirect scheme https if is_project_http url_project80_0 domain_project80_0`
(note the ending `_0`)

**Additional information you deem important (e.g. issue happens only occasionally):**

**Additional environment details (AWS, VirtualBox, physical, etc.):**
It was run on AWS
```
$ docker version
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Tue Mar 28 00:40:02 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Tue Mar 28 00:40:02 2017
 OS/Arch:      linux/amd64
 Experimental: false

$ docker image ls | grep vfarcic
vfarcic/docker-flow-proxy                                             latest              8e1e0074c769        5 days ago          38.9 MB
vfarcic/docker-flow-swarm-listener                                    latest              c75420b0ee33        2 weeks ago         21.6 MB
```

**Comment by vfarcic** [comment]
The good news is that this is a very easy fix thanks to your detailed description of the problem and config examples.  The bad news is that I'm on vacations with a very limited internet connection. I made a fix and could not test it properly. Can you do me a favor and try vfarcic/docker-flow-proxy:beta? If it works correctly, I'll promote it to an official release.

**Comment by vfarcic** [comment]
I managed to test it enough to be (more or less) confident that no other functionality broke (no side effects). It is released as tag `17.08.16-25`. I'd appreciate if you could have a look at it as well.

**Comment by ignacioalles** [comment]
Thank you very much,  Viktor! Tested with latest and it works great!
Sorry for bothering you on vacation!
